docs/data-collection/pre-session.md

```diff
@@ -138,21 +138,26 @@ Instructions of operations to be performed before the participant arrival, **bef
 - [ ] Go to room {{ secrets.rooms.projector | default("███") }}, where the projector is installed.
 - [ ] Switch the projector **ON** by hitting the power button, located on its right side.
 
-
-    !!! important "The projector's position SHALL NOT be modified unless it is evidently off"
-
-        Only in the case that the projector beam is not correctly targeting the projection screen inside the Scanning Room, which is possible if someone accidentally altered the configuration, you may adjust its position as follows:
-
-          - [ ] Adjust the projector tilt to center the projection if it does not properly aim the panel inside the scanner's bore.
-      E.g., change the height of the paper pile that supports it (see images, FENS papers).
-
+    
+    !!! important 
+        "The projector's position SHALL NOT be modified unless it is evidently off"
+    
+    Only in the case that the projector beam is not correctly targeting the projection screen inside the Scanning Room, which is possible if someone accidentally altered the configuration, you may adjust its position as follows:
+    
+        - [ ] Adjust the projector tilt to center the projection if it does not properly aim the panel inside the scanner's bore.
+           E.g., change the height of the paper pile that supports it (see images, FENS papers).
+        
```

Comment:
This is not equivalent to my suggestion. Perhaps this will render well?
```suggestion
    !!! important "The projector's position SHALL NOT be modified unless it is evidently off"

        Only in the case that the projector beam is not correctly targeting the projection screen inside the Scanning Room, which is possible if someone accidentally altered the configuration, you may adjust its position as follows:

        - [ ] Adjust the projector tilt to center the projection if it does not correctly aim the panel inside the scanner's bore.
           E.g., change the height of the paper pile that supports it (see images, FENS papers).

```